maven/blade.servicebuilder.test/src/main/java/com/liferay/blade/samples/servicebuilder/test/AddTestData.java

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
We should only add it one time, we don't need this AddTestData.java to all the projects, just to the liferay-workspace version.  That means we will need to modify replicateSources to avoid this file. Maybe we can add it to a new project under the liferay-workspace/tests/ folder so that it is automatically ignored by replicateSources.